docs/reference/release-policy.md

```diff
@@ -45,6 +45,14 @@ Whenever a new Camunda 8 version is released, we do our best to provide the new
 
 ![Console with notice to update the cluster in Camunda 8 SaaS](img/update-console.png)
 
+With the Camunda 8.5.0 release, the generation naming scheme in Camunda 8 will change and no longer include the patch version.
```

Comment:
🔧  I think the context is already SaaS but in first read I thought it would be good to highlight this only applies to SaaS
```suggestion
With the Camunda 8.5.0 release, the generation naming scheme in Camunda 8 SaaS will change and no longer include the patch version.
```